source/jormungandr/jormungandr/schedule.py

```diff
@@ -181,16 +184,24 @@ def _get_realtime_proxy(self, route_point):
             return None
         return rt_system
 
-    def _get_next_realtime_passages(self, rt_system, route_point, request):
+    def _get_next_realtime_passages(self, rt_system, route_point, request, **kwargs):
         log = logging.getLogger(__name__)
-
         next_rt_passages = None
+
+        try:
+            request.request_id = flask.request.id
```

Comment:
This won't probably work since request is a dict here, and even if it doesn't crash it won't do what we want.
In fact we dont't do any call to kraken in this greenlet so there is no point to set the `request_id` of a pbrequest.

What I wanted is here: http://flask.pocoo.org/docs/0.12/api/#flask.copy_current_request_context